modules/virtual_network_gateway/variables.tf

```diff
@@ -108,22 +86,37 @@ variable "generation" {
 
 variable "private_ip_address_enabled" {
   description = "Should private IP be enabled on this gateway for connections?"
```

Comment:
This should start with one sentence description, like: `Controls whether the private IP is enabled on the gateway.`

Reply:
fixed